### PR TITLE
S92.30 Add client settings

### PR DESCRIPTION
**Pull request by pvaass**

### CODE REVIEW

File: src/Connector/Client/ClientSettingsConnector.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php namespace ExpandOnline\KlipfolioApi\Connector\Client;
+
+use ExpandOnline\KlipfolioApi\Object\Client\ClientProperty;
+use ExpandOnline\KlipfolioApi\Connector\BaseApiResourceConnector;
+use ExpandOnline\KlipfolioApi\Object\Client\ClientResource;
+use ExpandOnline\KlipfolioApi\Object\Client\ClientSettings;
+
+/**
+ * Class ClientResourceConnector
```
Comment (mtricht) [inline src/Connector/Client/ClientSettingsConnector.php]: Wrong name.

**Comment by mtricht** [comment]
No tests?

**Comment by pvaass** [comment]
@swordbeta fixed

**Comment by pvaass** [comment]
test this please

**Comment by pvaass** [comment]
Test successful: https://dev.expandonline.nl/jenkins/job/klipfolio-api-php/85/console

**Pull request merged**